apps/web/src/server/applications/applications.router.js

```diff
@@ -35,6 +36,9 @@ router.use('/search-results', guards.assertDomainTypeExists, applicationsSearchR
 
 router.use('/create-new-case', guards.assertDomainTypeExists, applicationsCreateRouter);
 
+// TODO: not definitive url
```

Comment:
the code contains a series of pages under the url 'case/edit/{id}/{name of the page}'

these pages are not really part of this task and will be done/improved later. however I needed them to actually check the effectiveness of the refactor